lib/jobs.rb

```diff
@@ -27,8 +27,15 @@ def self.establish_connection!
     end
   end
 
-  def self.search(options)
-    @usajobs_api_connection.get(@endpoint, options).body if query_eligible?(options[:Keyword])
+  def self.job_scrub(keyword)
+    scrubbed_keyword = keyword.gsub(/#{JOB_RELATED_KEYWORDS}/, '').strip
+    keyword =~ /^#{JOB_RELATED_KEYWORDS}$/ ? keyword : scrubbed_keyword
```

Comment:
@peggles2 , can you explain a scenario where this 2nd line of code is necessary? It seems to me that this method could be reduced to simply: 
```ruby
keyword.gsub(/#{JOB_RELATED_KEYWORDS}/, '').squish
```

Reply:
If they search for just the word 'job' . we don't want to remove it. Then they would search for everything. So in that case we just search job.  But if we search for Nursing job, then we want to pass Nursing through.

Reply:
I added a comment there in case the next developer runs into the same question.

Reply:
I think it's preferable to strip those keywords regardless, because the postings don't always include those keywords. For example, if you search without any keywords, there are currently 7437 results. If you search with `Keyword=job`, that number drops to 3427.

Reply:
I think that the word "jobs" still returns results and you are now changing the behavior off the application.

Reply:
and its not just jobs internship and seasonal they search and get everything vs getting their results narrowed to catching the word internship

Reply:
if `Keyword=jobs` returns only 10% of the available jobs results, that's a problem. That could even reduce the results to zero depending on the location. If we want to scope by "internship" or "seasonal", the best way to do that would be with a followup story where queries with those terms passed the associated `PositionOfferingTypeCode` param (which we decided not to do until we see the need).

Reply:
Right but for now if they don't get enough results with jobs they have the see more jobs link. We can change this when we do the followup for internship.

Reply:
The purpose of this story is to ensure that we are returning jobs results as frequently as possible. Approximately a third of production job queries include "job" or "jobs", so we would be working against the purpose of this story if we continue to severely limit the cases where a jobs search returns results. With the code as it stands, we are supporting a very infrequent use case ("seasonal" or internship" are in < %5 of job queries) at the expense of the most frequent use case.